README.md

```diff
@@ -4,44 +4,27 @@
 [![IRC Channel](https://img.shields.io/badge/irc-%23calico-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#calico)
 # Project Calico
 
-Project Calico provides
-
-- A simple, pure layer 3 networking approach with no overlays for networking
-  "workloads" such as VMs and containers.
-- A distributed firewall implementing rich and flexible network policy,
-  imposed at ingress/egress to each workload.
-
-For more information see [the Project Calico website](http://www.projectcalico.org/learn/).
+<blockquote>
```

Comment:
A bit weird to use these tags in a markdown file, I think.

How about this:

```markdown
> Note that the documentation in this repo is targeted at Calico contributors.
> ## Documentation for Calico can be found at [docs.projectcalico.org](http://docs.projectcalico.org)
```

Formats like this:

> Note that the documentation in this repo is targeted at Calico contributors.
> ## Documentation for Calico can be found at [docs.projectcalico.org](docs.projectcalico.org)